stable/gocd/templates/gocd-agent-deployment.yaml

```diff
@@ -25,6 +25,10 @@ spec:
         app: {{ template "gocd.name" . }}
         release: {{ .Release.Name | quote }}
         component: agent
+      annotations:
```

Comment:
should we also consider setting these annotations on deployment instead of pod template?

Reply:
Hi, we didn't think of allowing annotations on deployment because the use case we were thinking about was to allow the pod to assume roles via the kube2iam library.